inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/AnnotatedListIdentifiers.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<!--
+ - Copyright 2017
+ - Ubiquitous Knowledge Processing (UKP) Lab
+ - Technische Universität Darmstadt
+ - 
+ - Licensed under the Apache License, Version 2.0 (the "License");
+ - you may not use this file except in compliance with the License.
+ - You may obtain a copy of the License at
+ -  
+ - http://www.apache.org/licenses/LICENSE-2.0
+ - 
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+-->
+
+<html xmlns:wicket="http://wicket.apache.org">
+<body>
+	<wicket:panel>
```

Comment:
Use 2 spaces for indenting XML/HTML files.

Reply:
Done